oauth.js

```diff
@@ -0,0 +1,30 @@
+const { loadSecrets } = require("./aws/ssm-helper");
+const config = require('config');
+
+const discordAuth = {
+  discordClientId: null,
+  discordClientSecret: null,
+  discordRedirectUrl: null,
+};
+
+async function initOAuth() {
```

Comment:
will we use OAuth for other integratoins, too? does it make sense to put it in an `oauth` directory and name the file discordAuth.js or something like this? 🤔

Reply:
Discussed and decided to keep as-is for now.